mitodl/mit-open
# Plain text news/events titles/authors; standardize html cleanup

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/4801

### Description (What does it do?)
- Unescapes characters for title and authors from Sloan news feed (doesn't seem necessary for others)
- Standardizes html cleanup by moving `learning_resources.etl.utils.clean_data` to `main.utils.clean_data` and using that for news_events ETL pipelines (summary, content).


### How can this be tested?
- Run `./manage.py backpopulate_news_events`
- Go to `http://api.open.odl.local:8063/api/v0/news_events/` and page through results.  There should be nothing like `&amp;`, `&gt;`, etc in titles or author names.  Fields that can contain HTML (summary, content) should have just tags that are included in `main.constants.ALLOWED_HTML_TAGS`.

### Additional context
The most relevant changes are in the `news_events` app, most everything else is a result of moving the `clean_data` function from `learning_resources.etl.utils` to `main.utils`
